README.ja.md

```
@@ -24,14 +24,27 @@
-curl -L https://github.com/trust-delta/tmai/releases/latest/download/tmai-$(uname -s | tr A-Z a-z)-$(uname -m).tar.gz | tar xz -C ~/.local/bin
+# 最新リリースを $HOME/.local (デフォルト prefix) に:
+curl -fsSL https://raw.githubusercontent.com/trust-delta/tmai/main/install.sh | bash
+
+# バージョンと prefix を指定:
+curl -fsSL https://raw.githubusercontent.com/trust-delta/tmai/main/install.sh \
+  | bash -s -- --version 2.0.0 --prefix /usr/local
+```
+
+展開先:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)